testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -2907,20 +2919,20 @@ QUnit.module('positioning', {
                 assert.strictEqual(newPosition.top, position.top - 100, 'top coordinate is correct');
             });
 
-            ['drag', 'resize'].forEach(moveMethodName => {
-                QUnit.module(moveMethodName, () => {
-                    QUnit.test('should not be changed after fullScreen is enabled and disabled', function(assert) {
```

Comment:
Let's leave this test to pin the researched case.
"should be restored to initial position after fullScreen is enabled and disabled".
I also suggest u to add a note:
"// NOTE: we should implement "pin" special button as a separate feature if we wan't keep drag/resize position"

Reply:
revert test